move_data_tags0.rb

```diff
@@ -0,0 +1,28 @@
+class MigrateTagsData < ActiveRecord::Migration
```

Comment:
To powinno się znajdować jako migracja - tak aby `rake db:migrate` automatycznie ją uruchamiało.